packages/esm-ward-app/src/ward-workspace/ward-patient-notes/form/notes-form.component.tsx

```diff
@@ -0,0 +1,171 @@
+          kind: 'success',
+          title: t('visitNoteSaved', 'Patient note saved'),
+        });
+      })
+      .catch((err) => {
+        createErrorHandler();
+
+        showSnackbar({
+          title: t('patientNoteSaveError', 'Error saving patient note'),
+          kind: 'error',
+          isLowContrast: false,
+          subtitle: err?.message,
+        });
+      })
+      .finally(() => {
+        setIsSubmitting(false);
+      });
+  };
+
+  const onError = (errors) => console.error(errors);
+
+  return (
+    <Form className={styles.form} onSubmit={handleSubmit(onSubmit, onError)}>
+      {errorFetchingEmrConfiguration && (
+        <div className={styles.formError}>
+          <InlineNotification
+            kind="error"
+            title={t('somePartsOfTheFormDidntLoad', "Some parts of the form didn't load")}
+            subtitle={t(
+              'fetchingEmrConfigurationFailed',
+              'Fetching EMR configuration failed. Try refreshing the page or contact your system administrator.',
+            )}
+            lowContrast
+            hideCloseButton
+          />
+        </div>
+      )}
+      <Stack className={styles.formContainer} gap={2}>
+        <Row className={styles.row}>
+          <Column sm={1}>
+            <span className={styles.columnLabel}>{t('note', 'Note')}</span>
+          </Column>
+          <Column sm={3}>
+            <Controller
+              name="wardClinicalNote"
+              control={control}
+              render={({ field: { onChange, onBlur, value } }) => (
+                <ResponsiveWrapper>
+                  <TextArea
+                    id="additionalNote"
+                    rows={rows}
+                    labelText={t('clinicalNoteLabel', 'Write your notes')}
+                    placeholder={t('wardClinicalNotePlaceholder', 'Write any notes here')}
+                    value={value}
+                    onBlur={onBlur}
+                    invalid={!!formState.errors.wardClinicalNote}
+                    invalidText={formState.errors.wardClinicalNote?.message}
+                    onChange={(event) => {
+                      onChange(event);
+                      const textareaLineHeight = 24; // This is the default line height for Carbon's TextArea component
+                      const newRows = Math.ceil(event.target.scrollHeight / textareaLineHeight);
+                      setRows(newRows);
+                    }}
+                  />
+                </ResponsiveWrapper>
+              )}
+            />
+          </Column>
+        </Row>
+      </Stack>
+      <Button
+        kind="primary"
+        className={styles.saveButton}
+        disabled={isSubmitting || isLoadingEmrConfiguration || errorFetchingEmrConfiguration}
+        type="submit">
+        {isSubmitting ? <InlineLoading description={t('saving', 'Saving...')} /> : <span>{t('save', 'Save')}</span>}
+      </Button>
```

Comment:
Don't we have a Cancel button in the workspace 🤔

Reply:
The design does not include one on this